pkg/zonemgr/zone_manager.go

```diff
@@ -28,52 +29,41 @@ func NewZoneManager() *ZoneManager {
 }
 
 func (zoneMgr *ZoneManager) init() {
-
-	fmt.Println("================ inside ZoneManager.init ===============")
+	zoneFileName := "/zones/db."
 
 	//todo fetch custom details
-	clusterName := "cluster"
-	zoneFileName := "/zones/db.secdns"
-
 	userAdminEmail := ""
-	userSubdomain := "domain.com"
+	userDomain := ""
 	userNameserverName := ""
 	userNameserverIP := ""
 
-	zoneMgr.zoneFileCopy = NewZoneFile(clusterName, userAdminEmail,
-		userNameserverName, userNameserverIP, userSubdomain)
+	zoneMgr.zoneFileCopy = NewZoneFile(userAdminEmail,
+		userNameserverName, userNameserverIP, userDomain)
 
 	zoneMgr.zoneFileCopy.init()
 
-	//fmt.Println("================ zoneMgr.zoneFileCopy.content ===============\n", zoneMgr.zoneFileCopy.content)
-
+	zoneFileName += zoneMgr.zoneFileCopy.domain
 	zoneMgr.zoneFileDns = NewDnsZoneFile(zoneFileName)
 	// todo return zoneFileDns.writeFile(zoneFileCopy.content) // is it necessary to create an empty zone file on load??
 }
 
 func (zoneMgr *ZoneManager) UpdateZone(namespacedName k8stypes.NamespacedName, interfaces []v1.VirtualMachineInstanceNetworkInterface) (err error) {
 
-	//fmt.Println("================ inside ZoneManager.UpdateZone ===============")
-	//fmt.Println("================ namespacedName: ===============", namespacedName)
-	//fmt.Println("================ interfaces: ===============", interfaces)
-
 	isUpdated, err := zoneMgr.zoneFileCopy.updateVMIRecords(namespacedName.Name, namespacedName.Namespace, interfaces)
 	if err != nil {
 		return err
 	}
 
-	//fmt.Println("================ zoneMgr.zoneFileCopy.content: ===============\n", zoneMgr.zoneFileCopy.content)
-	//fmt.Println("================ zoneMgr.zoneFileDns: ===============\n", zoneMgr.zoneFileDns)
-
 	//override on disk, create if not exist
 	if isUpdated {
 		err = zoneMgr.zoneFileDns.writeFile(zoneMgr.zoneFileCopy.content)
 		fmt.Println("********* err: ", err)
 
 		/////test
-		text, err := os.ReadFile("/zones/db.secdns")
+		text, err := os.ReadFile(zoneMgr.zoneFileDns.zoneFileFullName) //"/zones/db.cluster.secondary"
 		fmt.Println("********* err: ", err)
-		fmt.Println("********* text: ", string(text))
+		fmt.Println("********* zonefile name: ", zoneMgr.zoneFileDns.zoneFileFullName)
```

Comment:
Please don't print to the prompt.

Reply:
it's debug prints, temporary. I'll remove it